app/controllers/books_controller.rb

```diff
@@ -3,10 +3,12 @@ class BooksController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @books = Book.all
+    @books = Book.page(params[:page]).per(5)
   end
 
-  def show; end
+  def show
+    @reviews = @book.reviews.page(params[:page]).per(10)
```

Comment:
also this one you can follow https://github.com/giftnichakul/practice_ruby_on_rails/pull/11/files#r1223856866